src/layouts/ChatsLayout/components/TheCardGroups/index.vue

```diff
@@ -254,45 +307,177 @@ export default {
+        const filter = options.filter(
+          (queue) => !optionsQueues.includes(queue),
+        );
+
+        const selectedQueues = optionsQueues.map((queueUuid) => ({
+          uuid: queueUuid,
+          role: 1,
+        }));
+
+        const unselectedQueues = filter.map((queueUuid) => ({
+          uuid: queueUuid,
+          role: 2,
+        }));
+
+        const response = await Queues.editListQueues(
+          selectedQueues.concat(unselectedQueues),
+        );
+
+        callUnnnicAlert({
+          props: {
+            text: this.$t('chats.success_update_queues'),
+            type: 'success',
+          },
+          seconds: 5,
+        });
+
+        this.showModalQueue = false;
+
+        return response;
+      } catch (error) {
+        console.error(error);
+        callUnnnicAlert({
+          props: {
+            text: this.$t('chats.error_update_queues'),
+            type: 'error',
+          },
+          seconds: 5,
+        });
+        this.showModalQueue = false;
+      }
+    },
   },
 };
 </script>
-
 <style lang="scss" scoped>
 .container {
   flex: 1;
   display: flex;
   flex-direction: column;
   gap: $unnnic-spacing-stack-xs;
-
   .chat-groups {
     flex: 1 1;
-
     display: flex;
     flex-direction: column;
-
     margin-top: $unnnic-spacing-sm;
     padding-right: $unnnic-spacing-xs;
     margin-right: -$unnnic-spacing-xs; // For the scrollbar to stick to the edge
     overflow-y: auto;
     overflow-x: hidden;
-
     :deep(.unnnic-collapse) {
       padding-bottom: $unnnic-spacing-sm;
     }
-
     .no-results {
       color: $unnnic-color-neutral-cloudy;
       font-size: $unnnic-font-size-body-gt;
     }
   }
-
   .order-by {
     display: flex;
     justify-content: space-between;
-
+    gap: $unnnic-spacing-xs;
+    align-items: center;
     font-size: $unnnic-font-size-body-md;
     color: $unnnic-color-neutral-cloudy;
   }
+  .queue-modal {
+    .queue-modal-form {
+      display: grid;
+      gap: $unnnic-spacing-sm;
+      text-align: start;
+      .queue-modal-disclaimer {
+        display: flex;
+        flex-direction: row;
+        gap: $unnnic-spacing-xs;
+        justify-content: center;
+
+        padding: $unnnic-spacing-sm;
+
+        font-size: $unnnic-font-size-body-gt;
+        color: $unnnic-color-neutral-dark;
+
+        border-radius: 4px;
+        border: $unnnic-border-width-thin solid $unnnic-color-neutral-soft;
+      }
+      .queue-modal-select {
+        display: flex;
+        gap: $unnnic-spacing-xs;
+        .queue-modal-input {
+          flex: 1;
+        }
+      }
+    }
+    :deep(.unnnic-modal-container) {
+      .unnnic-modal-container-background {
+        width: 50%;
+        &-body-description-container {
+          padding-bottom: 0;
+        }
+      }
+    }
+    :deep(.unnnic-select-smart) {
+      .dropdown-data {
+        position: fixed !important;
+        top: inherit !important;
```

Comment:
Why are these `!important` needed here? using `!importart` is bad practice

Reply:
When I don't put `!import` the select component ends up looking like this in the modal:

![image](https://github.com/weni-ai/chats-webapp/assets/30897764/58bb7de3-cbe1-447e-9266-9c3271e38e22)

and the expected way it needs to be is this way:

![image](https://github.com/weni-ai/chats-webapp/assets/30897764/e1621fd6-71ce-460f-8f8b-e1e3e7fad2ba)

I haven't seen another way to resolve this yet 🧐

Reply:
in ModalBulkTransfer there is an example of how to do this without `!important`:

```
.modal-bulk-transfer {
  :deep(.unnnic-modal-container-background) {
    overflow: visible;
    
    .unnnic-modal-container-background-body {
      &-description,
      &-description-container {
        overflow: visible;
      }
    }
  }
```

Reply:
Thank you, that worked 🥳